public/app/features/provisioning/Wizard/PullStep.tsx

```diff
@@ -0,0 +1,22 @@
+import { useEffect } from 'react';
```

Comment:
Do we need a separate step for this or can this info be displayed elsewhere?

Reply:
I created a different step because we are still discussing if we want to trigger the initial sync synchronously as we do with the migrate step. So I was paving the group for @ryantxu to experiment on that. Backend will likely need changes for that.